src/tasks/helpers/replacers/UI5ElementClosestTo.ts

```diff
@@ -0,0 +1,151 @@
+import {Syntax} from "esprima";
+import * as ESTree from "estree";
+import * as recast from "recast";
+import {ASTReplaceable, NodePath} from "ui5-migration";
+
+const builders = recast.types.builders;
+
+type ParamArrayClosestTo = [ESTree.MemberExpression?, ESTree.Literal?];
+
+/**
+ * represents a module which exposes several functions such as
```

Comment:
remove this comment and add an appropriate one

Reply:
done